test/mod/test_fmt_v1.py

```diff
@@ -57,7 +57,7 @@
                     }
```

Comment:
(commit message `test/fmt/v1: use existing runner in manifests `)

You are quite consistent in using `relay`, I have to give you that!